client/my-sites/hosting/github/deployment-card/use-deployment-status.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery, UseQueryOptions } from 'react-query';
+import wp from 'calypso/lib/wp';
+
+export type DeploymentData = {
+	status: string;
+	move_failures: [];
+	remove_failures: [];
```

Comment:
We're not using the array contents right now but it's a good idea to have it typed already:

```suggestion
	move_failures: string[];
	remove_failures: string[];
```